server/src/sendgrid/sendgrid.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import * as SendGrid from '@sendgrid/mail';
+
+@Injectable()
+export class SendgridService {
+  constructor() {
+    SendGrid.setApiKey(process.env.SENDGRID_API_KEY);
+  }
+
+  async send(mail: SendGrid.MailDataRequired) {
+    const transport = await SendGrid.send(mail);
+    console.log(`E-Mail sent to ${mail.to}`);
```

Comment:
Thanks for logging this event. Can you use the NestJs `Logger` class, it will print other metadata. 

```ts
Logger.log(`E-Mail sent to ${mail.to}`, SendgridService.name);
```